.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: 'Create a report to help us improve Adventure '
+title: ""
+labels: 'type:bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+Please detail how you caused the bug
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+* If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via  https://imgur.com/ or similar.
+
+**Server (please complete the following information):**
```

Comment:
Since we'll be getting devs using this, it'd be better to ask for a link to maven pom or [gradle build scan](https://guides.gradle.org/creating-build-scans/) to get all their compile-time information at once, rather than asking for a single adventure version.

Reply:
I made adjustments...should be appropriate now